tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1114,6 +1099,10 @@ class CompletionSuite extends BaseCompletionSuite {
        |NoManifest scala.reflect
        |""".stripMargin,
     topLines = Option(3),
+    compat = Map(
+      "3" ->
+        ""
```

Comment:
Shouldn't we get `Number` from compiler completions here? Maybe in some cases we should also allow compler completions?

Reply:
Maybe we can allow compiler completions when we have typed bind i.e. `_: @@` or `abc`: xxx@@`?

Reply:
Also, should we move all the tests like `case xxx: So@@` from `CompletionSuite` to sth like `CasePatternSuite`, at least for Scala3, since we don't have this functionality for Scala2? I was thinking about creating new cross tests suite for case pattern completions both typed and untyped

Reply:
sure, makes sense to move out the tests